WcaOnRails/babel.config.js

```diff
@@ -51,12 +51,7 @@ module.exports = function(api) {
       '@babel/plugin-syntax-dynamic-import',
       isTestEnv && 'babel-plugin-dynamic-import-node',
       '@babel/plugin-transform-destructuring',
-      [
-        '@babel/plugin-proposal-class-properties',
-        {
-          loose: true
-        }
-      ],
+      '@babel/plugin-proposal-class-properties',
```

Comment:
This is unrelated to the issue but silences a warning during the asset compilation; I also have no idea why the `loose` here would be needed, it's there from the very beginning.